src/late_multicellular_stage/editor/MetaballEditorComponentBase.cs

```diff
@@ -241,7 +242,7 @@ public override void _Process(double delta)
             throw new InvalidOperationException($"{GetType().Name} not initialized");
 
         // TODO: should we display the hover metaballs setup on the previous frame here?
-        hoverMetaballsChanged = true;
+        // hoverMetaballsChanged = true;
```

Comment:
Can you explain the reasoning behind this change?

Reply:
I mostly added it not to update hover metaball each frame (back when meshes were updated in real-time), but it seems unneeded now.

Reply:
Alright, good to then revert this change so that it is easier to playtest this PR (to make the chance of there being some corner case bug here less likely).